### PR TITLE
use mystem histogram for some metrics

### PR DESCRIPTION
**Pull request by longbowlu**
## Description 

as title

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: crates/sui-core/src/authority_server.rs [hunk 1]
```diff
@@ -137,22 +135,18 @@ impl AuthorityServer {
...
     pub signature_errors: IntCounter,
-    pub tx_verification_latency: Histogram,
-    pub cert_verification_latency: Histogram,
-    pub consensus_latency: Histogram,
-    pub handle_transaction_latency: Histogram,
-    pub submit_certificate_consensus_latency: Histogram,
-    pub handle_certificate_consensus_latency: Histogram,
-    pub handle_certificate_non_consensus_latency: Histogram,
+    pub tx_verification_latency: MystenHistogram,
+    pub cert_verification_latency: MystenHistogram,
```
Comment (mwtian) [inline crates/sui-core/src/authority_server.rs:139]: Not sure if tx and cert verifications are suitable with Mysten Histogram. cc @andll

**Review by mwtian — APPROVED** [review]
LGTM to using Mysten Histogram for per RPC latencies.

File: crates/sui-core/src/authority_server.rs (hunk 1 above)
Comment (andll) [reply crates/sui-core/src/authority_server.rs:139]: I think this is ~ok, but I do wonder if even need a histogram for those metrics (isn't just sum/avg enough)?

Comment (longbowlu) [reply crates/sui-core/src/authority_server.rs:139]: landing this for now to catch the next release. Will do it in another PR if we decide to let them go

**Pull request merged**